py/mpconfig.h

```diff
@@ -636,6 +636,12 @@ typedef double mp_float_t;
 #define MICROPY_PY_DESCRIPTORS (0)
 #endif
 
+// Whether to support object (__getattr__, __delattr__ and __setattr__)
+// This costs some code size and makes all load attrs and store attrs slow
```

Comment:
'slow*er*' would be better wording imo, without measuring the actual impact it's hard to make absolute statements about it.